docs/user/intro/sphinx.rst

```diff
@@ -1,26 +1,107 @@
-Sphinx
-======
+Sphinx on Read the Docs
+=======================
+
+.. meta::
+   :description lang=en: Hosting Sphinx documentation on Read the Docs.
 
 Sphinx is a powerful documentation generator that
 has many features for writing technical documentation.
 
+Minimal configuration required to build an existing Sphinx project on Read the Docs looks like this,
+specifying a python 3.x toolchain on Ubuntu, defining the location of the installation requirements, and using the built-in
+:ref:`mkdocs <config-file/v2:sphinx>` command:
+
+
+.. TODO:: why do we install tree using apt_package here? Because it might not be in the requirements.txt but is still necessary?
+
 .. code-block:: yaml
    :caption: .readthedocs.yaml
 
-   version: 2
-   sphinx:
-      configuration: docs/conf.py
+      version: 2
+
+      build:
+         os: ubuntu-22.04
```

Comment:
`ubuntu-24.04`

Reply:
For this, and `tree` I'm using https://github.com/readthedocs/test-builds/blob/sphinx-7.0.x/.readthedocs.yaml -> is there something more recent I should be using?